homeassistant/components/zwave_js/config_flow.py

```diff
@@ -792,7 +792,9 @@ async def async_step_configure_addon(
                 CONF_ADDON_S2_AUTHENTICATED_KEY: self.s2_authenticated_key,
                 CONF_ADDON_S2_UNAUTHENTICATED_KEY: self.s2_unauthenticated_key,
                 CONF_ADDON_LOG_LEVEL: user_input[CONF_LOG_LEVEL],
-                CONF_ADDON_EMULATE_HARDWARE: user_input[CONF_EMULATE_HARDWARE],
+                CONF_ADDON_EMULATE_HARDWARE: user_input.get(
```

Comment:
This works, I did this as well initially but was looking for other places where it was missing

Reply:
The data schema for the form input already has a default for emulate hardware. The only place we call this step directly is when reverting the options to the old add-on options, if the re-configure flow fails for some reason. I think it's ok to add a default False when reverting though, if there's no existing value for emulate hardware.

Reply:
Side note: The re-configure flow is fragile, and I'm not really happy with how it works currently. I haven't concluded exactly how to do that yet, though.